cloud/google/machineactuator.go

```diff
@@ -130,6 +149,27 @@ func (gce *GCEClient) CreateMachineController(cluster *clusterv1.Cluster, initia
 		return err
 	}
 
+	// Create the configmap so the machine setup configs can be mounted into the node.
+	// TODO: create the configmap during bootstrapping instead of being buried in the machine actuator code.
+	machineSetupConfigs, err := gce.configWatch.ValidConfigs()
+	if err != nil {
+		return err
+	}
+	yaml, err := machineSetupConfigs.GetYaml()
+	if err != nil {
+		return err
+	}
+	configMap := corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{Name: "machine-setup"},
```

Comment:
This magic string is used in a few places. Is it possible to move it to a shared constant, and reference that instead?